backend/server/extraction_runnable.py

```diff
@@ -191,6 +192,11 @@ async def extract_entire_document(
         model_name=DEFAULT_MODEL,
     )
     texts = text_splitter.split_text(content)
+    if len(texts) > MAX_CHUNK_COUNT:
```

Comment:
@eyurtsev what's your opinion on
1. raising error as we do here;
2. truncating (e.g., proceed with first N chunks) and propagating information back to the user. if we did that, would we need to add metadata to the extraction response? lmk what you think (can do this in a separate PR too).